src/commands/archive-card.js

```diff
@@ -102,27 +102,43 @@ var __ = function(program, output, logger, config, trello, translator) {
             } else if (foundCards.length == 1) {
                 var card = foundCards[0];
                 if (options.title) {
-                    archiveCard(card);
+                    archiveCard(foundCards[0]);
                 } else {
-                    var list = new List({
-                        marker: ('›'.red) + " ",
-                        markerLength: 1
-                    });
-                    list.add(card, card.name);
-                    list.add(null, "[Cancel]");
-                    list.on('keypress', function(key, item) {
-                        switch (key.name) {
-                            case 'return':
-                                list.stop();
-                                archiveCard(item);
-                        }
-                    });
-                    list.start();
+                  //when would this else happen?
+                  // ? only one card found, but no title was given: this means the list specified has only one card
+                  // why would we even prompt the user?
+                    promptListSelection(foundCards);
```

Comment:
In this situation, I would still prompt the user as although there is only one card, they didn't specify a title so I would assume that they want to select one